INTERPRETING.md

```diff
@@ -0,0 +1,175 @@
+# Interpreting Test262 Tests
+
+All tests are declared as text files located within this project's `test`
+directory. In order to execute Test262 tests, runtimes must observe the
+following semantics.
+
+## Test Execution
+
+Test262 tests are only valid under the runtime environment conditions described
+here. Test environments may be further modified according to the metadata
+contained with each test--refer to the "Metadata" section for more details.
+
+### Realm Isolation
+
+Each test must be executed in a new [ECMAScript
+realm](https://tc39.github.io/ecma262/#sec-code-realms) dedicated to that test.
+Unless configured otherwise (via the `module` flag), source text must be
+interpreted as [global
+code](http://www.ecma-international.org/ecma-262/6.0/#sec-types-of-source-code).
+
+### Test262-Defined Bindings
+
+The contents of the following files must be evaluated in the test realm's
+global scope prior to test execution:
+
+1. `harness/assert.js`
+2. `harness/sta.js`
+
+### Host-Defined Functions
+
+The following values must be defined as writable, configurable, non-enumerable
+properties of the global scope prior to test execution.
+
+- **`print`** A function that exposes the string value of its first argument to
+  the test runner. This is used as a communication mechanism for asynchronous
+  tests (via the `async` flag, described below).
+
+### Strict Mode
+
+Unless configured otherwise (via the `noStrict`, `onlyStrict`, or `raw` flags),
+each test must be executed twice: once in ECMAScript's non-strict mode, and
+again in ECMAScript's strict mode.
```

Comment:
@jmdyck I'm glad you're digging this! I've added some text to be more explicit
about enabling strict mode. I've taken a cue from your comments in gh-299,
refraining from referring to the input as "source text" since (as you
mentioned) that term has a specific meaning in this context.

> To run in strict mode, the test contents must be modified prior to execution--[a "use strict" directive](https://tc39.github.io/ecma262/#sec-directive-prologues-and-the-use-strict-directive) must be inserted as the initial character sequence of the file, followed by a semicolon (`;`) and newline character (`\n`):
>
>     "use strict";
>
> This must precede any additional text modifications described by test metadata.

What do you think?